src/batou_ext/tests/test_configure.py

```diff
@@ -172,4 +172,11 @@ def test_prepare(root, mocker, component, tmpdir):
         instance.script = "/srv/s-myuser/start-application.sh"
         instance.watchdog_script_path = "/srv/s-myuser/watchdog-wrapper.py"
         instance.healthcheck_url = "https://example.com"
+    elif component_name == "batou_ext.mail.Mailpit":
+        batou_ext.http.HTTPBasicAuth(
+            env_name="restricted-area",
+            username="AzureDiamond",
+            password="hunter2",
+            basic_auth_string="AzureDiamond:$apr1$kialsve4$uP1QDkMw6Tia5YFgi4U7v1",
```

Comment:
for these defaults you can use _required_params e.g. https://github.com/flyingcircusio/batou_ext/blob/d5bb15393b31b78351152e2b3fc5a13b85f02ba1/src/batou_ext/s3.py#L40

Reply:
Fair, updated.